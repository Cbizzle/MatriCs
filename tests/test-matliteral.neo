int main() {
	int[2,2,2,2] a;
	int i;
	int j;
	int k;
	int l;

	a = [[[[1,2], [3,3]],
		 [[7,2], [9,1]]],
		 [[[1,2], [3,3]],
		 [[7,2], [9,1]]]];

	for (i = 0; i < 2; i = i+1) {
		for (j = 0; j < 2; j = j+1) {
			for (k = 0; k < 2; k = k+1) {
				for (l = 0; l < 2; l = l+1) {
					print_int(a[i,j,k,l]);
				} 
			}
		}
	}
	return 0;
}