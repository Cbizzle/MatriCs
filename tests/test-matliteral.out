1233729112337291